src/base/types.ts

```diff
@@ -171,30 +171,35 @@ export interface PayingAccountFees {
   };
 }
 
+/**
+ * Unsigned transaction data in JSON a format
+ */
 export interface TransactionPayload {
   /**
    * This is what a Polkadot signer ".signPayload" method expects
+   *
```

Comment:
I found this note a little confusing. The submission of the signed transaction now accepts either `payload`, `rawPayload` or `TransactionPayload`. If the note was specific to when submitting maybe change to

> @note this field is recommended to be passed in with the signature when submitting a signed transaction